### ISSUE TITLE
Problems with the model

### ISSUE THREAD

**Issue by meritakete22** [MCG-NJU/MultiSports#28]
When i'm trying to make a simple inference with one video of the dataset with the model slowfast or slowonly given by mmaction2 for multisport i have an issue with the test_pipeline, that is needed to make an inference but the models doesnt have and give me this error:
![image](https://github.com/MCG-NJU/MultiSports/assets/124732141/deadfd2e-601b-4280-ba46-113c48b83631)
I searched how to solve this problem and I saw that you can use the basic pipelane as test_pipeline:
![image](https://github.com/MCG-NJU/MultiSports/assets/124732141/e67a580e-a9ae-402b-82df-4bf77b1a6ac3)
But when it goes to make the transform for SampleAVAFrames it brings me a new error with the tags of results:
![image](https://github.com/MCG-NJU/MultiSports/assets/124732141/8b3cd398-9c4a-4531-a7dd-1d6ff4bea08e)
I have no idea what to do now.
I apreciaty any help

**Comment by Abdulkadir19997** [comment]
Hi, I am stuck in here too, did you find any solution for that?

**Comment by meritakete22** [comment]
No, it was for my TFG and I decided to change the dataset to AVA
